src/components/Login/index.tsx

```diff
@@ -50,12 +75,18 @@ function LoginModalContent(props: Props) {
         <ModalCloseButton role="closeButton" />
         <ModalBody>
           <Flex flexDir="column" justifyContent="space-around" gap="25px" paddingX="20px">
-            <Button variant="loginBtn" position="relative" ref={initialRef}>
+            <Button
+              onClick={() => handleLogin()}
```

Comment:
if you don't pass any  props, you can simples use "onClick={handleLogin}" 
you don't need need to change it,  just  a reminder